UPDATING.md

```diff
@@ -52,6 +52,12 @@ assists users migrating to a new version.
 
 ## Master
 
+### SparkJDBCHook default connection
+
+For SparkJDBCHook default connection was `spark-default`, and for SparkSubmitHook it was
+`spark_default`. Both hooks now use the `spark_default` which is a common pattern for the connection
+names used across all providers.
```

Comment:
Do we need upgrade-check for change like this?

Reply:
Likely yes!